README.md

```diff
@@ -35,66 +35,74 @@ yarn add @untemps/react-vocal
 import Vocal from '@untemps/react-vocal'
 
 const App = () => {
-	const [result, setResult] = useState('')
-
-	const _onVocalStart = () => {
-		setResult('')
-	}
-
-	const _onVocalResult = (result) => {
-		setResult(result)
-	}
-
-	return (
-		<div className="App">
-			<span style={{ position: 'relative' }}>
-				<Vocal
-					onStart={_onVocalStart}
-					onResult={_onVocalResult}
-					style={{ width: 16, position: 'absolute', right: 10, top: -2 }}
-				/>
-				<input defaultValue={result} style={{ width: 300, height: 40 }} />
-			</span>
-		</div>
-	)
+   const [result, setResult] = useState('')
+   
+   const _onVocalStart = () => {
+      setResult('')
+   }
+   
+   const _onVocalResult = (result) => {
+      setResult(result)
+   }
+   
+   return (
+      <div className="App">
+         <span style={{ position: 'relative' }}>
+            <Vocal
+               onStart={_onVocalStart}
+               onResult={_onVocalResult}
+               style={{ width: 16, position: 'absolute', right: 10, top: -2 }}
+            />
+            <input defaultValue={result} style={{ width: 300, height: 40 }} />
+         </span>
+      </div>
+   )
 }
 ```
 
 #### Custom component
 
-By default, `Vocal` displays an icon with two states (idle/listening):
+By default, `Vocal` displays an icon with two states:
 
-![Idle icon](assets/icon-idle.png)
-![Listening icon](assets/icon-listening.png)
+- Idle  
+![Idle state](assets/icon-idle.png)
+- Listening  
+![Listening state](assets/icon-listening.png)
 
-But you can provide your own component:
+But you can provide your own component.  
+- With a simple React element:
 
 ```javascript
 import Vocal from '@untemps/react-vocal'
 
 const App = () => {
-	const [isListening, setIsListening] = useState('')
+   return (
+      <Vocal>
+         <button>Start</button>
+      </Vocal>
+   )
+}
+```
+In this case, a `onClick` handler is automatically attached to the component to start a recognition session.
```

Comment:
Et s'il y a deux boutons ou un bouton plus bas dans la hiérarchie ?

Reply:
The `onClick` handler is attached to the root tag

Reply:
Proposed addition to the doc:

> Only the first direct descendant of `Vocal` will receive the `onClick` handler. If you want to use a more complex hierarchy, use the function syntax below.